### PR TITLE
Guidance on adding bullet points to a placeholder when using CSV upload

### PR DESCRIPTION
**Pull request by saimaghafoor**
Adding guidance on how to personalise bullet points via a spreadsheet. Users ask how to do this through support. Guidance for adding bullets via API will be added to the the API docs. Story on trello: https://trello.com/c/NtSFH9Cu

### CODE REVIEW

File: app/templates/views/guidance/using-notify/formatting.html [hunk 1]
```diff
@@ -60,6 +66,26 @@ <h3 class="heading-small" id="bullets">Bullet points</h3>
 <p class="govuk-body">There’s more <a class="govuk-link govuk-link--no-visited-state"
     href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps">guidance about bullet points on GOV.UK</a>.</p>
 
+<p class="govuk-body">Copy this example to personalise the bullet points each time you send a message:</p>
```
Comment (CrystalPea) [inline app/templates/views/guidance/using-notify/formatting.html:69]:
```suggestion
<p class="govuk-body">Copy this example to add personalised bullet points each time you send a message:</p>
```

File: app/templates/partials/templates/guidance-formatting.html [hunk 2]
```diff
@@ -22,6 +22,20 @@ <h3 class="heading-small">Bullet points</h3>
...
+
+<p class="govuk-body">Copy this example to personalise bullet points each time you send a message:</p>
+<pre class="formatting-example"><code class="lang-md">Introduce bullet points with a lead-in line ending in a colon:
+
+((bullet points))
+
+  Leave one empty line space before the next paragraph.
+</code></pre>
+
+<p class="govuk-body">Your spreadsheet of recipient details should include a column for bullet points.</p>
```
Comment (CrystalPea) [inline app/templates/partials/templates/guidance-formatting.html:34]:
```suggestion
<p class="govuk-body">Your spreadsheet of recipient details should include a column for each bullet point.</p>
```

Comment (karlchillmaid) [reply app/templates/partials/templates/guidance-formatting.html:34]: I’m not sure about this suggested change.

The spreadsheet needs a column headed `bullet points`.
In that column, each individual cell needs to include the bullets for that recipient. One recipient per row.

I understand ‘a column for each bullet point’ to mean something different.

**Review by karlchillmaid — APPROVED** [review]
Content looks good.

(This is not a dev/code review)

File: app/templates/partials/templates/guidance-bullet-points.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    </p>
+
+    <p class="govuk-body">This is an example of a spreadsheet that will create personalised bullet points:</p>
+
+    <pre class="formatting-example"><code class="lang-md">
+    email address, bullet points, bullet points, bullet points
+    recipient-email@example.com, item 1, item 2, item 3
+    recipient2-email@example.com, item 1, item 2
+
+    </code></pre>
```
Comment (karlchillmaid) [inline app/templates/partials/templates/guidance-bullet-points.html:57]: I’m not sure we need this.

File: app/templates/partials/templates/guidance-bullet-points.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+        Leave one empty line space before the next paragraph.
+    </code></pre>
+
+    <p class="govuk-body">Your spreadsheet of recipient details should include a column for each of the bullet points. </p>
+
+    <p class="govuk-body">
+        Create a column with the placeholder name as a title for each bullet point your list will have.
+        Then add one list item per column.
+    </p>
```
[the remark this answers is not in the document]
Comment (CrystalPea) [reply app/templates/partials/templates/guidance-bullet-points.html:48]: No, they don't.

File: app/templates/partials/templates/guidance-bullet-points.html (hunk 3 above)
Comment (CrystalPea) [reply app/templates/partials/templates/guidance-bullet-points.html:57]: ok, let's get rid of the CSV example.

File: app/templates/partials/templates/guidance-bullet-points.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+</code></pre>
+
+<p class="govuk-body">To create sub-items, add an indent of 2 spaces before the asterisk or dash.</p>
+<p class="govuk-body">There’s more <a class="govuk-link govuk-link--no-visited-state"
+        href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps">guidance about bullet
+        points on GOV.UK</a>.</p>
+
+<!-- PERSONALISED BULLET POINTS -->
+{% set personalisedBullets %}
+    <p class="govuk-body">Copy this example to add personalised bullet points each time you send a message:</p>
```
Comment (karlchillmaid) [inline app/templates/partials/templates/guidance-bullet-points.html:33]:
```suggestion
    <p class="govuk-body">Copy this example to add a <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_personalisation') }}">placeholder</a> to your message template:</p>
```

File: app/templates/partials/templates/guidance-bullet-points.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+    <pre class="formatting-example"><code class="lang-md">
+        Introduce bullet points with a lead-in line ending in a colon:
+
+        ((bullet points))
+
+        Leave one empty line space before the next paragraph.
+    </code></pre>
+
+    <p class="govuk-body">Your spreadsheet of recipient details should include a column for each of the bullet points. </p>
```
Comment (karlchillmaid) [inline app/templates/partials/templates/guidance-bullet-points.html:43]:
```suggestion
    <p class="govuk-body">To send the message, <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_bulk_sending') }}">upload a list of recipient details</a>.</p>
```

File: app/templates/partials/templates/guidance-bullet-points.html (hunk 4 above)
Comment (karlchillmaid) [inline app/templates/partials/templates/guidance-bullet-points.html:48]:
```suggestion
    <p class="govuk-body">Your spreadsheet should include one column for each bullet point. The column names must match the placeholder in the template.</p>
    <p class="govuk-body">Notify will fill in the placeholder with your bullet points.</p>
```

File: app/templates/partials/templates/guidance-bullet-points.html (hunk 3 above)
Comment (karlchillmaid) [reply app/templates/partials/templates/guidance-bullet-points.html:57]:
```suggestion
```

**Pull request merged**